modules/thoughtleadrBidAdapter.js

```diff
@@ -1,29 +1,66 @@
-'use strict';
 var bidfactory = require('src/bidfactory');
 var bidmanager = require('src/bidmanager');
 var utils = require('src/utils');
-var adloader_1 = require('src/adloader');
+var ajax_1 = require('src/ajax');
 var adaptermanager = require('src/adaptermanager');
-var ROOT_URL = '//cdn.thoughtleadr.com/v4/';
+
+var COOKIE_SYNC_ID = 'tldr-cookie-sync-div';
+var UID_KEY = 'tldr_uid';
+var URL_API = 'tldr' in window && tldr.config.root_url ? tldr.config.root_url : '//a.thoughtleadr.com/v4/';
+var URL_CDN = 'tldr' in window && tldr.config.cdn_url ? tldr.config.cdn_url : '//cdn.thoughtleadr.com/v4/';
 var BID_AVAILABLE = 1;
+var BID_UNAVAILABLE = 2;
+
+function getVal(key) {
+  if ('localStorage' in window) {
```

Comment:
Looks like this feature test doesn't work so well. See [this gist](https://gist.github.com/paulirish/5558557) for some examples, and [MDN's suggestion](https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API#Testing_for_availability) for a more rigorous test.